scripts/subscribeEventProof.js

```diff
@@ -0,0 +1,151 @@
+
+const { Api } = require('@cennznet/api');
+require("dotenv").config();
+const logger = require('./logger');
+const nodemailer = require('nodemailer');
+
+// Send email when accounts eth balance is less than gas fees
+const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+        user: process.env.supportEmail,
+        pass: process.env.pass
+    }
+});
+
+const mailOptions = {
+    from: process.env.supportEmail,
+    to: process.env.supportEmail,
+    subject: 'Please top up eth balance',
+    text: 'To keep the validator relayer running, topup the eth account'
```

Comment:
```suggestion
    text: 'To keep the validator relayer running, topup the eth account'
```

can we add the address in this message?